features/dd-sdk-android-webview/src/main/kotlin/com/datadog/android/webview/internal/log/WebViewLogEventConsumer.kt

```diff
@@ -19,18 +20,23 @@ import com.google.gson.JsonObject
 internal class WebViewLogEventConsumer(
     private val sdkCore: FeatureSdkCore,
     internal val userLogsWriter: DataWriter<JsonObject>,
-    private val rumContextProvider: WebViewRumEventContextProvider
+    private val rumContextProvider: WebViewRumEventContextProvider,
+    val sampleRate: Float
```

Comment:
It can be `private` it seems.

```suggestion
    private val sampleRate: Float
```